### PR TITLE
Rename fastq file service

### PR DESCRIPTION
**Pull request by seallard**
## Description
Let's give the service a better name indicating what it is doing.

### CODE REVIEW

File: cg/models/cg_config.py [hunk 1]
```diff
@@ -589,8 +591,8 @@ def trailblazer_api(self) -> TrailblazerAPI:
         return api
 
     @property
-    def fastq_file_service(self) -> FastqFileService:
-        return FastqFileService()
+    def fastq_file_service(self) -> FastqConcatenationService:
```
Comment (ChrOertlin) [inline cg/models/cg_config.py:594]:
```suggestion
    def fastq_concat_service(self) -> FastqConcatenationService:
```

File: cg/meta/delivery/delivery.py [hunk 2]
```diff
@@ -26,12 +28,12 @@ class DeliveryAPI:
     def __init__(
         self,
         delivery_path: Path,
-        fastq_file_service: FastqFileService,
+        fastq_file_service: FastqConcatenationService,
```
Comment (ChrOertlin) [inline cg/meta/delivery/delivery.py:31]:
```suggestion
        fastq_fconcat_service: FastqConcatenationService,
```

File: cg/meta/delivery/delivery.py [hunk 3]
```diff
@@ -26,12 +28,12 @@ class DeliveryAPI:
...
         housekeeper_api: HousekeeperAPI,
         store: Store,
     ):
         self.delivery_path: Path = delivery_path
-        self.fastq_file_service: FastqFileService = fastq_file_service
+        self.fastq_file_service: FastqConcatenationService = fastq_file_service
```
Comment (ChrOertlin) [inline cg/meta/delivery/delivery.py:36]:
```suggestion
        self.fastq_concat_service: FastqConcatenationService = fastq_concat_service
```

**Review by diitaz93 — APPROVED** [review]
👍

**Review by ChrOertlin — APPROVED** [review]
Looks good, just make sure to also rename parameters from `fastq_file_service` to `fastq_concat_service` (or something)

File: cg/meta/delivery/delivery.py (hunk 3 above)
Comment (seallard) [reply cg/meta/delivery/delivery.py]: https://github.com/Clinical-Genomics/cg/pull/3216/commits/1ff7c8a292b4abd6d9a29b1040002ccdc1ab41ff

File: cg/meta/delivery/delivery.py (hunk 2 above)
Comment (seallard) [reply cg/meta/delivery/delivery.py]: https://github.com/Clinical-Genomics/cg/pull/3216/commits/1ff7c8a292b4abd6d9a29b1040002ccdc1ab41ff

File: cg/models/cg_config.py (hunk 1 above)
Comment (seallard) [reply cg/models/cg_config.py]: https://github.com/Clinical-Genomics/cg/pull/3216/commits/1ff7c8a292b4abd6d9a29b1040002ccdc1ab41ff

**Pull request merged**